src/site/content/en/blog/constructable-stylesheets/index.md

```diff
@@ -61,10 +61,11 @@ preload stylesheets without injecting them into the DOM.
 ## Constructing a stylesheet
 
 Rather than introducing a new API to accomplish this, the [Constructable
-StyleSheets](https://wicg.github.io/construct-stylesheets) specification makes
-it possible to create stylesheets imperatively by invoking the `CSSStyleSheet()`
-constructor. The resulting CSSStyleSheet object has two new methods that make it
-safer to add and update stylesheet rules without triggering [Flash of Unstyled
+StyleSheets](https://drafts.csswg.org/cssom/#dom-cssstylesheet-cssstylesheet)
```

Comment:
I think this has landed now, right?

https://www.w3.org/TR/cssom-1/#dom-cssstylesheet-cssstylesheet